Source/Spatializer/CrossPlatform/SpatializerMixerPlugin.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Please note that this will only work on Unity 5.2 or higher.
+
+#include "AudioPluginUtil.h"
+#include "HrtfWrapper.h"
+#include "VectorMath.h"
+#include "mathutility.h"
+
+#include <cstring>
+
+namespace SpatialMixer
+{
+    struct EffectData
+    {
+        // History buffer used when the DSP buffer is smaller than the HRTF quantum
+        std::unique_ptr<float[]> HrtfHistoryBuffer;
+
+        // Current read offset into the history buffer
+        int ReadOffset;
+    };
+
+    int InternalRegisterEffectDefinition(UnityAudioEffectDefinition&)
```

Comment:
I think you can just delete this

Reply:
Looks like I still need it, get linker error if I remove it.

Reply:
Aha! This might explain why you didn't see a behavior difference by not calling it in the spatializer plugin. Looks like Unity is expecting to call this itself.